tests/test-coreutils/src/poll.spec.ts

```diff
@@ -236,7 +236,9 @@ describe('Poll', () => {
         expect(ticker.join(' ')).to.equal(tocker.join(' '));
         poll.tick.then(tock).catch(() => undefined);
       };
-      await poll.tick.then(tock);
+      // Kick off the promise listener, but void its settlement to verify that
```

Comment:
This version of the test will only pass with this fix, so the comment about voiding the promise instead of awaiting it indicates that this is by design.